subhub/tests/unit/customer/test_customer_calls.py

```diff
@@ -19,6 +19,17 @@ class MockCustomer:
     object = "customer"
     subscriptions = [{"data": "somedata"}]
 
+    def props(self, cls):
+        return [i for i in cls.__dict__.keys() if i[:1] != "_"]
+
+    def get(self, key, default=None):
+        properties = self.props(MockCustomer)
+        logger.info("properties", properties=properties)
+        if key in properties:
+            return properties["key"]
```

Comment:
does this work? quoted "key" instead of the variable `key`? Maybe I don't understand what is going on here.